vm/src/vm/errors/hint_errors.rs

```diff
@@ -184,6 +184,12 @@ pub enum HintError {
     NPairBitsTooLowM,
     #[error("{0}")]
     SyscallError(Box<str>),
+    #[error("excess_balance_func: Failed to fetch {0} dictionary. It is either missing or has unexpected data types")]
+    ExcessBalanceFailedToFecthDict(Box<str>),
```

Comment:
ExcessBalanceFailedToFecthDict -> ExcessBalanceFailedToFetchDict

Small typo in Fetch